projects/demo/src/modules/services/breakpoint/examples/1/component.ts

```diff
@@ -0,0 +1,19 @@
+import {Component, Inject} from '@angular/core';
+import {changeDetection} from '@demo/emulate/change-detection';
+import {encapsulation} from '@demo/emulate/encapsulation';
+import {TuiBreakpointService} from '@taiga-ui/core';
+
+@Component({
+    selector: `tui-breakpoint-example`,
+    templateUrl: `./template.html`,
+    styleUrls: [`./style.less`],
+    changeDetection,
+    encapsulation,
+    providers: [TuiBreakpointService],
```

Comment:
There's no need to add it to `providers`, since it is `providedIn: 'root'`. Also please remove it from the `Setup` tab.